radio/src/gui/colorlcd/widgets/modelbmp.cpp

```diff
@@ -34,6 +34,20 @@ class ModelBitmapWidget: public Widget
 
     void refresh(BitmapBuffer * dc) override
     {
+      std::string filename = std::string(g_model.header.bitmap);
+
+      // set font colour from options[0]
+      lcdSetColor(persistentData->options[0].value.unsignedValue);
+
+      // get font size from options[1]
+      LcdFlags fontsize = persistentData->options[1].value.unsignedValue << 8u;
```

Comment:
Can you please check if there is an existing macro for this? I believe there should be.